tools/windows/DatadogAgentInstaller/CustomActions/ServiceCustomAction.cs

```diff
@@ -47,32 +47,19 @@ public ServiceCustomAction(ISession session)
         {
         }
 
-        private static ActionResult EnsureNpmServiceDepdendency(ISession session)
+        private static ActionResult EnsureNpmServiceDependendency(ISession session)
         {
             try
             {
-                var addLocal = session.Property("ADDLOCAL").ToUpper();
-                session.Log($"ADDLOCAL={addLocal}");
                 using var systemProbeDef = Registry.LocalMachine.OpenSubKey(@"SYSTEM\CurrentControlSet\Services\datadog-system-probe", true);
                 if (systemProbeDef != null)
                 {
-                    if (string.IsNullOrEmpty(addLocal))
+                    // Remove the dependency between "datadog-system-probe" and "ddnpm" services since
+                    // the "datadogagent" service now takes care of starting the "ddnpm" service.
```

Comment:
This is minor because it is in a comment, I'm mentioning it to make sure there isn't code somewhere that is expecting this behavior:

> the "datadogagent" service now takes care of starting the "ddnpm" service.

the "datadogagent" service doesn't start "ddnpm", "datadog-system-probe" handles starting / stopping "ddnpm"